README.md

```diff
@@ -8,22 +8,46 @@
 
 Initialise service
 
-    service = BingAdsReporting::Service.new({developerToken: '', applicationToken: '', authenticationToken: '', :accountId: '', customerId: ''})
+- Ad Insight Service
+
+```
+    service = BingAdsReporting::AdInsightService.new({developerToken: '', applicationToken: '', authenticationToken: '', :accountId: '', customerId: ''})
```

Comment:
Maybe for later (or now if you have time) but it looks like both of these classes are coupled with the data structure passed in the argument.

I think for more maintainability we should decouple it into its own class (or struct) as it seems these always come together.

Reply:
later I think